components/mailchimp/sources/new-campaign/new-campaign.js

```diff
@@ -0,0 +1,101 @@
+const common = require("../common/timer-based");
```

Comment:
To include `constants.mjs`
```suggestion
const common = require("../common/timer-based");
const constants = require("../constants.mjs");
// or import constants from "../constants.mjs";
```